src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -72,7 +89,10 @@ const ReportActionContextMenu = (props) => {
                 <ReportActionContextMenuItem
                     icon={contextAction.icon}
                     text={contextAction.text}
+                    successIcon={contextAction.successIcon}
+                    successText={contextAction.successText}
                     isMini={props.isMini}
+                    onPress={() => Str.result(contextAction.onPress, props.reportAction)}
```

Comment:
I found this confusing. All the other `contextActions` will soon have an `onPress`, so let's just give all the others `onPress: () => {}` for now. That way we don't need to do `Str.result` (which is also included in underscore, so I'm not sure why we bother having it in `expensify-common`).

Reply:
I used it from `expensify-common` as this one supports arguments. Ok. Will add onPress: () => {}.